### PR TITLE
docs: update the landing page

### PR DESCRIPTION
**Pull request by tianzhou**
Based on the recent discussion with the marketing consultant. Please check the preview for details.

### CODE REVIEW

File: components/LogoCloudSection.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  >
+    <img
+      class="col-span-2 max-h-20 w-full object-contain lg:col-span-1"
+      src="~/assets/logo/aptive-logo.webp"
+      alt="aptive"
+    />
+    <img
+      class="col-span-2 max-h-16 -mt-2 w-full object-contain lg:col-span-1"
+      src="~/assets/logo/dingding-logo.webp"
+      alt="kakao"
```
Comment (boojack) [inline components/LogoCloudSection.vue:13]:
```suggestion
      alt="dingding"
```

File: components/LogoCloudSection.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    />
+    <img
+      class="col-span-2 max-h-16 w-full object-contain lg:col-span-1"
+      src="~/assets/logo/evermos-logo.webp"
+      alt="kakao"
```
Comment (boojack) [inline components/LogoCloudSection.vue:18]:
```suggestion
      alt="evermos"
```

File: components/LogoCloudSection.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<template>
+  <div
```
Comment (boojack) [inline components/LogoCloudSection.vue:2]: Cannot get the relationship between these company and Bytebase. Maybe add a title to this section?

image

File: components/LogoCloudSection.vue [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    />
+    <img
+      class="col-span-2 max-h-16 w-full object-contain lg:col-span-1"
+      src="~/assets/logo/kakao-logo.webp"
+      alt="kakao"
+    />
+  </div>
+</template>
+
+<script lang="ts">
```
Comment (boojack) [inline components/LogoCloudSection.vue:33]: The entire script tag is not used.

**Review by boojack — CHANGES_REQUESTED** [review]

File: components/LogoCloudSection.vue (hunk 4 above)
Comment (tianzhou) [reply components/LogoCloudSection.vue]: Done

File: components/LogoCloudSection.vue (hunk 3 above)
Comment (tianzhou) [reply components/LogoCloudSection.vue:2]: per offline discussion

**Comment by tianzhou** [comment]
PTAL

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**